commit 2113b6b
Author: Adar Dembo <adar@cloudera.com>
Date:   Sun May 26 14:39:46 2019 -0700

    Makefile.am: fixes for building against older glibc
    
    In older versions of glibc (such as the version found on el6 machines),
    the clock_gettime function is in librt rather than in libc directly. Our
    bundled jemalloc depends on clock_gettime and may link against librt, so
    let's make sure the memkind library links against librt too.

diff --git a/Makefile.am b/Makefile.am
index f791ce7..de57d90 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -44,7 +44,7 @@ libmemkind_la_SOURCES = src/hbwmalloc.c \
 
 
 libmemkind_la_LIBADD = jemalloc/obj/lib/libjemalloc_pic.a
-libmemkind_la_LDFLAGS = -version-info 0:1:0 -ldl
+libmemkind_la_LDFLAGS = -version-info 0:1:0 -ldl -lrt
 include_HEADERS = include/hbw_allocator.h \
                   include/hbwmalloc.h \
                   include/memkind.h \
